app/src/main/java/com/imdb_compose/MyApplication.kt

```diff
@@ -0,0 +1,47 @@
+package com.imdb_compose
+
+import android.app.Application
+import coil.ImageLoader
+import coil.ImageLoaderFactory
+import coil.disk.DiskCache
+import coil.memory.MemoryCache
+import coil.request.CachePolicy
+import coil.util.DebugLogger
+import dagger.hilt.android.HiltAndroidApp
+import okhttp3.Call
+import okhttp3.OkHttpClient
+
+@HiltAndroidApp
+class MyApplication: Application(), ImageLoaderFactory {
+    override fun newImageLoader(): ImageLoader {
+        val okHttpClient = OkHttpClient.Builder().build()
```

Comment:
We should inject this instance of `okHttpClient`

https://stackoverflow.com/a/68825889